ROADMAP.md

```diff
@@ -12,56 +12,57 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
 
 * Update project governance policies [#1445](https://github.com/rook/rook/issues/1445)
 * Add Core Infrastructure Initiative (CII) Best Practices [#1440](https://github.com/rook/rook/issues/1440)
-* Integrate a more robust controller framework (e.g., CoreOS Operator SDK or Kubebuilder) [#1981](https://github.com/rook/rook/issues/1981)
 * Build and integration testing improvements
   * Increase PR quality gates (e.g., [vendoring verification](https://github.com/rook/rook/issues/1822), [license scanning](https://github.com/rook/rook/issues/1993), etc.)
-  * Update promotion and release channels to align with storage provider specific statuses [#1885](https://github.com/rook/rook/issues/1885)
-  * Refactor test framework and helpers to support multiple storage providers [#1788](https://github.com/rook/rook/issues/1788)
-  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
-  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
-* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
-* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
-* Support for dynamic provisioning of new storage types
-  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
-  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
 * New storage providers
   * NFS operator and CRDs (backed by arbitrary PVs) [#1551](https://github.com/rook/rook/issues/1551)
-  * Cassandra design [#1910](https://github.com/rook/rook/issues/1910)
-* CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
-  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
-* Minio
-  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+  * Cassandra operator [#1910](https://github.com/rook/rook/issues/1910)
+  * Nexenta EdgeFS operator [#1532](https://github.com/rook/rook/issues/1532)
 * Ceph
+  * Declare Ceph CRDs to be stable v1
   * Added support for Mimic and later versions in addition to Luminous [#2004](https://github.com/rook/rook/issues/2004)
   * Automated upgrade support (initial) [#997](https://github.com/rook/rook/issues/997)
-  * Manage an existing Ceph cluster (basic) [#1868](https://github.com/rook/rook/issues/1868)
+  * CSI plug-in documentation [#2234](https://github.com/rook/rook/pull/2234)
   * OSDs
     * Run on arbitrary PVs (local storage) as an alternative to host path [#796](https://github.com/rook/rook/issues/796) [#919](https://github.com/rook/rook/issues/919)
```

Comment:
arbitrary PVs won't be plumbed through Ceph, right? just the common rook.io types are being updated I believe? @galexrt

Reply:
Yes, so #2309 is merged and each storage backend will get a PR.
I have done Minio and CockroachDB in one PR as they were pretty similar, see #2318.
PVCs for usage with Ceph will probably be implemented in the original PR #2132, after #2318 is finished.